Build failure if we bump to go1.26 in go.mod

```
pkg/metrics/collector/kernel_metrics_test.go:46:38: (*testing.common).Errorf format %+q has arg got of wrong type map[string]uint64
pkg/metrics/collector/kernel_metrics_test.go:65:35: (*testing.common).Errorf format %+q has arg got of wrong type map[string]uint64
pkg/metrics/collector/netlink_metrics_test.go:46:24: (*testing.common).Fatalf format %+q has arg statMap of wrong type map[*k8s.io/api/core/v1.Pod]github.com/GoogleCloudPlatform/netd/pkg/metrics/collector.netlinkStats
```

--- a/pkg/metrics/collector/kernel_metrics_test.go
+++ b/pkg/metrics/collector/kernel_metrics_test.go
@@ -43,7 +43,7 @@ TcpExt: 0 0 0 0 0 440 99 21800 3`
 		t.Errorf("failed parseNetstat, got %q", err)
 	}
 	if !mapContainsAll(got, want) {
-		t.Errorf("parseNetstatVals returns %+q, want %+q", got, want)
+		t.Errorf("parseNetstatVals returns %+v, want %+v", got, want)
 	}
 }
 
@@ -62,7 +62,7 @@ UdpLite: 0 0 0 0 0 0 0 0`
 		t.Errorf("failed parseNetstat, got %q", err)
 	}
 	if !mapContainsAll(got, want) {
-		t.Errorf("parseSnmpVals returns %+q, want %+q", got, want)
+		t.Errorf("parseSnmpVals returns %+v, want %+v", got, want)
 	}
 
 }

--- a/pkg/metrics/collector/netlink_metrics_test.go
+++ b/pkg/metrics/collector/netlink_metrics_test.go
@@ -43,7 +43,7 @@ func TestCreateStatMap(t *testing.T) {
 	statMap := createStatMap(snapshots)
 	want := map[*v1.Pod]netlinkStats{myPod: {retransmits: 8, tcpConnections: 2}}
 	if statMap[myPod] != want[myPod] {
-		t.Fatalf("Fatal, got %+q, wanted %+q", statMap, want)
+		t.Fatalf("Fatal, got %+v, wanted %+v", statMap, want)
 	}
 }
 

